docs/Makefile

```
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
+# You can set these variables from the command line.
+SPHINXOPTS    =
+SPHINXBUILD   = python -msphinx
```

```suggestion
SPHINXBUILD   = python -m sphinx
```
For clarity of flag and flag input.